src/Magnum/BulletIntegration/DebugDraw.h

```diff
@@ -0,0 +1,177 @@
+
+#include "Magnum/BulletIntegration/Integration.h"
+
+namespace Magnum { namespace BulletIntegration {
+
+/**
+@brief Bullet physics debug visualization
+
+This class implements @ref btIDebugDraw, which allows rendering a visualization
+of a bullet physics world for debugging purposes.
+
+## Usage
+
+@code
+btDynamicsWorld* btWorld = // ...
+DebugDraw debugDraw;
+
+debugDraw.setDebugMode(DebugDraw::Mode::DrawWireframe | DebugDraw::Mode::DrawConstraints);
+btWorld->setDebugDrawer(&debugDraw);
+
+// Per frame call:
+
+debugDraw.setViewProjectionMatrix(projection*view);
+btWorld->debugDrawWorld();
+@endcode
+*/
+class MAGNUM_BULLETINTEGRATION_EXPORT DebugDraw: public btIDebugDraw {
+    public:
+
+        /** @brief Debug mode enum */
+        enum class Mode: Int {
+            NoDebug = DBG_NoDebug,                      /**< Disable debug rendering */
+            DrawWireframe = DBG_DrawWireframe,          /**< Draw wireframe of all collision shapes */
+            DrawAabb = DBG_DrawAabb,                    /**< Draw axis aligned bounding box of all collision object */
+            DrawFeaturesText = DBG_DrawFeaturesText,    /**< Draw text for features */
+            DrawContactPoints = DBG_DrawContactPoints,  /**< Draw contact points */
+            NoDeactivation = DBG_NoDeactivation,        /**< Disable deactivation of objects */
+            NoHelpText = DBG_NoHelpText,                /**< Diable help text */
+            DrawText = DBG_DrawText,                    /**< Enable text drawing */
+            ProfileTimings = DBG_ProfileTimings,        /**< Profile timings */
+            EnableSatComparison = DBG_EnableSatComparison, /**< Enable Sat Comparison */
+            DisableBulletLCP = DBG_DisableBulletLCP,    /**< Disable Bullet LCP */
+            EnableCCD = DBG_EnableCCD,                  /**< Enable CCD */
+            DrawConstraints = DBG_DrawConstraints,      /**< Draw constaints */
+            DrawConstraintLimits = DBG_DrawConstraintLimits, /**< Draw constraint limits */
+            FastWirefram = DBG_FastWireframe,           /**< Draw fast wireframes */
+            DrawNormals = DBG_DrawNormals,              /**< Draw normals */
+            DrawFrames = DBG_DrawFrames,                /**< Draw frames */
+        };
+
+        typedef Containers::EnumSet<Mode> Modes;
+
+        /**
+         * @brief Constructor
+         * @param initialBufferCapacity Amount of lines for which to reserve
+         *        memory in the buffer vector.
+         *
+         * Sets up @ref Shaders::VertexColor3D, @ref Buffer and @ref Mesh for physics debug rendering.
+         */
+        DebugDraw(UnsignedInt initialBufferCapacity=0):
+            btIDebugDraw(),
+            _mesh{MeshPrimitive::Lines}
+        {
+            _mesh.addVertexBuffer(_buffer, 0, Shaders::VertexColor3D::Position{}, Shaders::VertexColor3D::Color{});
+            _bufferData.reserve(initialBufferCapacity*4);
+        }
```

Comment:
So... instead of providing a default value for `initialCapacity` parameter, right?